src/main/java/com/github/kmu_shell_we/domain/season/_team/controller/TeamController.java

```diff
@@ -35,9 +37,11 @@ public ApiResponse<TeamListResponse> getTeams(@Parameter(description = "시즌 I
 
     @GetMapping("/me")
     @Operation(summary = "내 팀 조회")
-    public ApiResponse<GetTeamResponse> getMyTeam(@Parameter(description = "시즌 ID") @PathVariable UUID seasonId) {
-
-        return ApiResponse.ok(teamService.getMyTeam(seasonId));
+    public ApiResponse<GetTeamResponse> getMyTeam(
+            @Parameter(description = "시즌 ID") @PathVariable UUID seasonId,
+            @AuthenticationPrincipal User user
+    ) {
```

Comment:
경로 변수보다 유저가 조금 더 중요하다고 생각해서 첫번째 인자로 옮기는 것이 어떨까요?

```suggestion
    public ApiResponse<GetTeamResponse> getMyTeam(
            @AuthenticationPrincipal User user,
            @Parameter(description = "시즌 ID") @PathVariable UUID seasonId
    ) {
```